content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -178,7 +178,7 @@ A time zone database from the Go standard library is included in the binaries an
 
 ### Unsupported TimeZone specification
 
-The implementation of the CronJob API in Kubernetes {{< skew currentVersion >}} lets you set
+The implementation of the CronJob API in Kubernetes {{< skew currentPatchVersion >}} lets you set
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.